ocaml/src/asd_server.ml

```diff
@@ -1249,7 +1251,13 @@ let execute_update : type req res.
                   (function
                     | Lwt.Canceled -> Lwt.fail Lwt.Canceled
                     | ConcurrentModification -> Lwt.return `Retry
-                    | exn -> Lwt.fail exn) >>= function
+                    | NegDiskUsage ->
+                       Lwt_log.fatal "negative disk usage detected." >>= fun () ->
+                       Lwt_extra2.flush_logging () >>= fun () ->
+                       let () = exit 666 in
+                       Lwt.return `Retry
```

Comment:
`exit 666` can just be the last statement .. doesn't the compiler warn this is dead code?